test/modules/fundingManager/bondingCurve/abstracts/RedeemingBondingCurveBase_v1.t.sol

```diff
@@ -537,7 +576,14 @@ contract RedeemingBondingCurveBaseV1Test is ModuleTest {
         _workflowFee = bound(_workflowFee, 0, _bps - 1);
         vm.assume(_collateralFee + _workflowFee < _bps);
 
-        _depositAmount = bound(_depositAmount, 1, 1e38);
+        _depositAmount = bound(
+            _depositAmount,
+            helperCalculateMinDepositAmount(
+                _collateralFee, _issuanceFee, _collateralFee
+            ),
+            1e38
+        );
+        emit hm(_depositAmount);
```

Comment:
hm

Reply:
Yeah so testing is still WIP, and will prob need that event still there. Hope you didn't spend to much time looking at the tests. Added in the PR description that those are still todos